k9mail-library/src/main/java/com/fsck/k9/mail/internet/Viewable.java

```diff
@@ -42,8 +42,14 @@ public Text(Part part) {
     }
 
     class Flowed extends Textual {
-        public Flowed(Part part) {
+        private boolean isDelsp;
```

Comment:
The field name and parameter name should not contain the `is` prefix. It is only used in getter methods that return a boolean value, i.e. `isDelSp()` rather than `getDelSp()`.

Please also be consistent with the naming, i.e. don't use both `delsp` and `delSp`. The RFC uses "DelSp", so we should go with `delSp` to be consistent with Java naming conventions.